HelpSource/Classes/String.schelp

```diff
@@ -120,6 +120,8 @@ code::
 
 method::==
 Returns a link::Classes/Boolean:: whether the two Strings are equal.
+
+Overridden by the need to check for equality of each instance of link::Classes/Char:: with its integer index as independent of identity. See link::Classes/Object#Equality and Identity::.
```

Comment:
I don't understand what you are trying to say. I'd just add the link to equality and identity

Reply:
> Overridden by the need to check for equality of each instance of link::Classes/Char:: with its integer index as independent of identity. See link::Classes/Object#Equality and Identity::.

This part is a rephrase of the following part in the `Object` class:

https://github.com/supercollider/supercollider/blob/60b291ab18bb21c868748d8f93e5b357c435813a/HelpSource/Classes/Object.schelp#L207

How about this?
```
This method overrides the same method of link::Classes/Object#-==:: by the need to check for equality of each instance of link::Classes/Char:: with its integer index as independent of identity. See link::Classes/Object#Equality and Identity::.
```

Reply:
I'd delete this then. The existing description is fine.

You are explaining the implementation of String's == method. This isn't needed and means we can't change the code without changing the documentation (which we'd forget to do), in general, its best to just stick to how the method is used and the signature.

Reply:
Thanks, OK!